guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc

```diff
@@ -1,34 +1,39 @@
 [id="Enrolling_Server_with_the_AD_Server_{context}"]
 = Enrolling {ProjectServer} with the AD server
 
-In the {Project} CLI, enroll {ProjectServer} with the Active Directory server.
+When using Active Directory (AD) as an external authentication source for {Project}, use GSS-Proxy for enhanced security.
+The traditional process of Kerberos authentication in Apache requires the Apache process to have read access to the keytab file.
+GSS-Proxy allows you to implement stricter privilege separation for the Apache server by removing access to the keytab file while preserving Kerberos authentication functionality.
 
 .Prerequisites
-* GSS-proxy and {nfs-client-package} are installed.
-+
-Installing GSS-proxy and {nfs-client-package}:
+* GSS-Proxy and {nfs-client-package} are installed on your {ProjectServer}.
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {project-package-install} gssproxy {nfs-client-package}
 ----
+* Your AD user account has administrator privileges.
 
 .Procedure
+On your {ProjectServer}:
+
 . Install the required packages:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-install} sssd adcli realmd ipa-python-compat krb5-workstation samba-common-tools
+# {project-package-install} adcli ipa-python-compat krb5-workstation oddjob oddjob-mkhomedir realmd samba-common-tools sssd
```

Comment:
Confirmed in https://issues.redhat.com/browse/SAT-26355 and in [RHEL AD integration docs](https://issues.redhat.com/browse/SAT-26355).